README.md

```diff
@@ -124,29 +126,26 @@ The easiest way to tweak react-colorful is to create another stylesheet to overr
 As you probably noticed the color picker itself doesn't include an input field, but don't worry if you need one. **react-colorful** is a modular library that allows you to build any picker you need. Since `v2.1` we provide an additional component that works perfectly in pair with our color picker.
 
 <details>
-  <summary>How to use <code>HexInput</code></summary><br />
+  <summary>How to use <code>HexColorInput</code></summary><br />
 
-```diff
-import ColorPicker from "react-colorful";
-+import HexInput from "react-colorful/HexInput";
+```js
+import { HexColorPicker, HexColorInput } from "react-colorful";
 import "react-colorful/dist/index.css";
```

Comment:
I believe the CSS import is now incorrect because I started to mess with it before realizing we'd need something beyond Microbundle more than likely to do what I had imagined. Could just revert that though.

Reply:
Sorry, not sure that I got what you mean here.

Reply:
I moved the CSS into `src/css/...`, so the output is now at `dist/css/index.css`. Will need fixing/reverting/something.

However, I still think it could be nice to, as an after-build process, move the CSS up a level, so `dist` and `css` will both be top-level for this package. This would allow a user to import CSS like `import "react-colorful/css/index.css"` which I think would be a bit nicer.

Reply:
Does it really? Didn't know that. Would've assumed it kept the same directory structure. (Sorry, can't check right now as still traveling). In that case, less valuable. 

But sure, no reason to do anything further here on that.

Reply:
Actually we could even copy that to the root. Then it will be available by:
```
import "react-colorful/index.css"
or 
import "react-colorful/styles.css"
```

Anyway, I think this is topic for another discussion.

Reply:
Yeah so long as we're just copying it doesn't need to be a breaking change and can be done elsewhere at some other time.